### PR TITLE
update docs for 7.4.0

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

File: docs/plugins/inputs/file.asciidoc [hunk 1]
```diff
@@ -380,7 +380,7 @@ The sincedb record now has a last active timestamp associated with it.
 If no changes are detected in a tracked file in the last N days its sincedb
 tracking record expires and will not be persisted.
 This option helps protect against the inode recycling problem.
-Filebeat has a {filebeat-ref}/faq.html[FAQ about inode recycling].
+Filebeat has a {filebeat-ref}/faq.html#inode-reuse-issue[FAQ about inode recycling].
```
Comment (jsvd) [inline docs/plugins/inputs/file.asciidoc]: @karenzone this seems to be causing the docs build to fail, need your help here :)

Comment (karenzone) [reply docs/plugins/inputs/file.asciidoc]: run elasticsearch-ci/docs

Comment (karenzone) [reply docs/plugins/inputs/file.asciidoc]: @elasticmachine, run elasticsearch-ci/docs

Comment (karenzone) [reply docs/plugins/inputs/file.asciidoc]: Note to self (@karenzone):  Everything looks good with the exception of the four plugins I didn't publish before this update was created. I think we should merge this PR, and then immediately follow up with another docgen to pick up the latest.

**Pull request merged**